.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: node_js
-node_js: "8"
+node_js:
```

Comment:
Regarding the node versions, don't you think it would be better to explicitly list the versions we want. Here the tests are running twice for node 8, matched by `- 8` and `lts/*`, and once for node 10, but none for node 9. So I think it would be better to have
```yml
- 8
- 9
- 10
```

Reply:
Sure, can do! Node 11 is coming out later this month, so it will switch to latest (11), LTS (10) and 8, that's why I added 8 after manually. I'll remove 8, it can be added later when 11 comes out